app/scripts/modules/core/src/search/infrastructure/SearchResult.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+
+import { NgReact } from 'core/reactShims';
+import { IRecentHistoryEntry } from 'core/history';
+
+export interface ISearchResultProps {
+  item: IRecentHistoryEntry & { account?: string; displayName?: string };
```

Comment:
Is this a quick way to extend a class?

Reply:
This is an intersection type and in this case yes it's a quick way to extend an interface.  Happy to switch to `extends` if you prefer

Reply:
I like this, just wanted to make sure I was reading it correctly